src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -91,20 +97,32 @@
 
     private PodTemplate template;
 
+    @CheckForNull
+    private KubernetesSlave slave;
+
     public PodTemplateBuilder(PodTemplate template) {
         this.template = template;
     }
 
+    public PodTemplateBuilder withSlave(KubernetesSlave slave) {
+        this.slave = slave;
+        return this;
+    }
+
+    @Deprecated
```

Comment:
Why is a new deprecated method being added?

Reply:
this one is not new, the new one is `public Pod build()`